i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/virtualization-hyperv-2012-nrpe.md

```diff
@@ -5,80 +5,601 @@ title: Hyper-V 2012 NSClient++ NRPE
+
+</TabItem>
+<TabItem value="Scvmm-Vm-Status" label="Scvmm-Vm-Status">
+
+| Macro             | Description                                                                                                                                      | Valeur par défaut                    | Obligatoire |
+|:------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------|:-----------:|
+| FILTERVM          | Filter virtual machines (can be a regexp)                                                                                                        |                                      |             |
+| FILTERDESCRIPTION | Filter by description (can be a regexp)                                                                                                          |                                      |             |
+| FILTERHOSTGROUP   | Filter hostgroup (can be a regexp)                                                                                                               |                                      |             |
+| CRITICALSTATUS    | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{vm}, %{status}, %{hostgroup}                | not %{status} =~ /Running\|Stopped/i |             |
+| WARNINGSTATUS     | Define the conditions to match for the status to be WARNING . You can use the following variables: %{vm}, %{status}, %{hostgroup}                |                                      |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). | --verbose                            |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib64/nagios/plugins// -H 10.0.0.1 -p  -t   -c check_centreon_plugins -a 'apps::microsoft::hyperv::2012::local::plugin' 'scvmm-vm-status'  '  \
+	--scvmm-hostname="" \
+	--scvmm-username="" \
+	--scvmm-password="" \
```

Comment:
mandatory?

Reply:
Il n'est pas indiqué dans le template d'host que c'est mandatory mais je connais mal l'outil c'est peut-être le cas et le script l'a mal identifié en générant la doc. Pareil je veux bien l'avis de @omercier

Reply:
The hostname is not mandatory but both username, password and port are

Reply:
Username and password are not in the host template) so not shown as mandatory explicitly.
@omercier